.github/workflows/build.yml

```diff
@@ -185,3 +185,55 @@ jobs:
     - run: make clang-debug
     - name: tests (wasm2c tests excluding memory64)
       run: ./test/run-tests.py wasm2c --exclude-dir memory64
+
+  build-arch:
```

Comment:
How about calling this `build-s390x`?   Maybe `build-arch` harks of Arch linux?

Reply:
hmm, maybe `build-cross`?

Reply:
Sure! sgtm

Reply:
done!